### ISSUE TITLE
docker inspect -f '{{ .NetworkSettings.IPAddress }}' ${CONTAINER}

### ISSUE THREAD

**Issue by changemenemo** [warehouseman/xubuntu-x2go-phusion#3]
docker inspect -f '{{ .NetworkSettings.IPAddress }}' ${CONTAINER} 
not correct anymore, sometimes it display it in some container and sometimes not

**Comment by changemenemo** [comment]
the way to do it is now : 
`docker inspect -f '{{.Name}} - {{.NetworkSettings.IPAddress }}' $(docker ps -aq)`

**Comment by martinhbramwell** [comment]
So you have tested that and it works well for you?

**Comment by changemenemo** [comment]
yes and not the old one, could you look into it when you have time?

**Comment by martinhbramwell** [comment]
Look into??   To you just mean for me to change the README, to reflect what you suggest, or is there something more you want me to do?

**Comment by changemenemo** [comment]
sorry for the delay again :)
yeah just change the readme I can push some changes if you want :)?
